bug: tests failing due to timezone issues

### Describe the bug

The tests are currently failing locally, indicating a potential problem with the timezone settings. 
The specific test in question is `TestMustToDate` and its subtest `TestDate`. The error message suggests that the expected and actual times are not equal, with the main difference being the timezone offset (+0000 UTC vs +0200 CEST).

### To Reproduce

1. Run the tests locally (offseting of UTC)
2. Observe the failure of the `TestMustToDate` test

### Expected behavior

The tests should pass without any errors or failures.

### Code Snippet

```Shell
--- FAIL: TestMustToDate (0.00s)
    --- FAIL: TestMustToDate/TestDate (0.00s)
        conversion_functions_test.go:143:
            	Error Trace:	/Users/andig/htdocs/sprout/test_helpers_test.go:57
            	Error:      	Not equal:
            	            	expected: "time.Time-2024-05-09 00:00:00 +0000 UTC"
            	            	actual  : "time.Time-2024-05-09 00:00:00 +0200 CEST"
```
```


### Version of software

v0.4.0

### Additional context

This issue is likely related to recent changes in the formatting codebase, which were triggered by running gofmt. Further investigation is needed to determine the root cause of this problem.

See: https://github.com/go-sprout/sprout/pull/38#issuecomment-2159949706

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/conversion_functions_test.go
+++ b/conversion_functions_test.go
@@ -135,7 +135,7 @@ func TestToDuration(t *testing.T) {
 
 func TestMustToDate(t *testing.T) {
 	var tests = mustTestCases{
-		{testCase{"TestDate", `{{$v := mustToDate "2006-01-02" .V }}{{typeOf $v}}-{{$v}}`, "time.Time-2024-05-09 00:00:00 +0000 UTC", map[string]any{"V": "2024-05-09"}}, ""},
+		{testCase{"TestDate", `{{$v := mustToDate "2006-01-02 15:04:05 MST" .V }}{{typeOf $v}}-{{$v}}`, "time.Time-2024-05-09 00:00:00 +0000 UTC", map[string]any{"V": "2024-05-09 00:00:00 UTC"}}, ""},
 		{testCase{"TestInvalidValue", `{{$v := mustToDate "2006-01-02" .V }}{{typeOf $v}}-{{$v}}`, "", map[string]any{"V": ""}}, "cannot parse \"\" as \"2006\""},
 		{testCase{"TestInvalidLayout", `{{$v := mustToDate "invalid" .V }}{{typeOf $v}}-{{$v}}`, "", map[string]any{"V": "2024-05-09"}}, "cannot parse \"2024-05-09\" as \"invalid\""},
 	}

